airflow/contrib/operators/azure_container_instances_operator.py

```diff
@@ -18,6 +18,9 @@
 # under the License.
 
 from time import sleep
+from collections import namedtuple
+
+from typing import Dict, Sequence
```

Comment:
Frankly I'm not very familiar with `mypy`, so can not give useful comment here.

A minor question here: why the packages are not imported in alphabetic order? And given `typing` is a standard library since Py3.5, we can group these three (`collections`, `time`, `typing`) together?

Reference: https://github.com/reddit-archive/reddit/wiki/PythonImportGuidelines

Reply:
Understand this practice is not really followed very well in this project along the way. But we can start to follow it in the new code added at least.

Reply:
Updated. We might want to consider https://github.com/timothycrosley/isort or similar to do this consistently.